examples/bme280_normal_mode.py

```diff
@@ -8,27 +8,27 @@
 """
 import time
 import board
-import adafruit_bme280
+from adafruit_bme280 import advanced
```

Comment:
What do you think about this? It would allow the other code to stay the same and be more readable.
```suggestion
import adafruit_bme280.advanced as adafruit_bme280
```

Reply:
Does this work ok? You'll need to update the rest of the file and the other examples too.

Reply:
@tannewt will need to test and correct all the examples. Will do for sure, sorry I did not mentioned yesterday. Thanks